package.json

```diff
@@ -24,29 +23,25 @@
     "@babel/core": "^7",
     "@babel/eslint-parser": "^7",
     "@open-wc/testing": "^3",
+    "@rollup/plugin-node-resolve": "^13",
     "@web/test-runner": "^0.13",
     "eslint": "^8",
     "eslint-config-brightspace": "^0.18",
     "eslint-plugin-html": "^6",
     "eslint-plugin-import": "^2",
     "eslint-plugin-lit": "^1",
     "eslint-plugin-sort-class-members": "^1",
-    "rollup": "^0.67.1",
-    "rollup-plugin-commonjs": "^9.2.0",
-    "rollup-plugin-json": "^3.1.0",
```

Comment:
JSON plugin wasn't used and CommonJS plugin isn't needed since `frau-jwt` is ESM now.